CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Add optional `level_of_authentication` parameter to Account API `get_sign_in_url`.
 * Store response body in new `http_body` field of HTTP exceptions
 * Added Pact tests for the `LinkCheckerApi` adapters.
+* Added Pact tests for the `AccountApi` adapters.
```

Comment:
The interesting change here is adding the `get_attributes_names` method, not that we also added a test for it